wallet/wallet.go

```diff
@@ -2975,6 +2976,32 @@ func (w *Wallet) StakeInfo() (*StakeInfoData, error) {
 			}
 		}
 
+		expiredTicketNum := 0
+		revokeTicketPtrs := make([]*chainhash.Hash, len(revokedTickets))
+		for i := range revokedTickets {
+			revokeTicketPtrs[i] = &revokedTickets[i]
+		}
+
+		// Check the expired ticket pool for the presense of tickets.
+		expiredBitSetBStr, err := chainClient.ExistsExpiredTickets(revokeTicketPtrs)
```

Comment:
I guess that's a good point, I was using a inclusive/exclusive view of them.  All expired are missed, but not all missed are expired.